cpp/Doxyfile.in

```diff
@@ -737,7 +738,7 @@ WARN_NO_PARAMDOC       = YES
 # a warning is encountered.
 # The default value is: NO.
 
-WARN_AS_ERROR          = YES
+WARN_AS_ERROR          = NO
```

Comment:
why are turning off warnings as errors?

Reply:
This PR is still a WIP and there is one pesky warning I cant seem to get around:

```cpp/include/cuml/common/logger.hpp:152: warning: argument 'unnamed' of command @param is not found in the argument list of ML::Logger::log(int level, const char *fmt,...)```

The DoxyDoxygen has a bug with `...` variable arguments and keeps reintroducing the `unnamed` argument.